src/quo2/components/settings/data_item/view.cljs

```diff
@@ -0,0 +1,91 @@
+(ns quo2.components.settings.data-item.view
+  (:require [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.icon :as icons]
+            [quo2.components.settings.data-item.style :as style]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.foundations.resources :as quo2.resources]
+            [react-native.fast-image :as fast-image]
+            [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.components.list-items.preview-list :as preview-list]
+            [quo2.components.common.not-implemented :as not-implemented]))
+
+(defn- render-right-side
+  [label icon-right? icon-color]
+  [rn/view
+   {:style style/right-container}
+   (case label
+     :preview [preview-list/preview-list
+               {:type      :user
+                :size      24
+                :list-size 3}]
+     :graph   [text/text "graph"]
```

Comment:
Is this correct without a translation?

Reply:
Yes, it's a simple placeholder for now since the graph is not implemented yet.